worlds/tunic/er_scripts.py

```diff
@@ -93,7 +94,18 @@ def place_event_items(world: "TunicWorld", regions: Dict[str, Region]) -> None:
         region.locations.append(location)
 
 
-def vanilla_portals() -> Dict[Portal, Portal]:
+# all shops are the same shop. however, you cannot get to all shops from the same shop entrance.
+# so, we need a bunch of shop regions that connect to the actual shop, but the actual shop cannot connect back
+def create_shop_region(world: "TunicWorld", regions: Dict[str, Region]) -> None:
```

Comment:
Clever 🔥

Reply:
What would happen if you made entrances to the shop a one way door since it is a dead end? I imagine it doesn't have any logical pathing _through_ it but there could be some glitches/tricks I'm not aware of